gnpy/core/node.py

```diff
@@ -26,7 +26,7 @@ def __new__(cls, latitude=0, longitude=0, city=None, region=None):
         return super().__new__(cls, latitude, longitude, city, region)
 
 class Node:
-    def __init__(self, uid, name=None, params=None, metadata=None, operational=None):
+    def __init__(self, uid, name=None, params=None, metadata=None, operational=None, type_variety=None):
```

Comment:
What is the motivation for introducing `type_variety` here?

Reply:
In the current version type_variety is duplicated: it is present as an attribute of Fiber and of Fiber.params. We want to get rid of this redundancy; we believe that as type_variety is a description rather than a physical parameter of the fiber, the right place should be outside of Fiber.params. Moreover, we think that the right way to include this attribute in any element is to define it in the Node constructor.